CHANGELOG.md

```diff
@@ -74,8 +76,19 @@ attached to all deprecated names to discourage their use.
 * In `Data.Nat.Properties`:
   ```agda
   +-*-suc ↦ *-suc
+
+  ```
+
+* In `Data.Bin.Properties`:
+  ```agda
+  2[1+-≢0  ↦  2[1+x]≢0
+  1+[2-≢0  ↦  1+2[1+x]≢0
+
   ```
 
+  In several property names ``2, 2*`` replaced with ``double``.
```

Comment:
Again doesn't need to be listed in the CHANGELOG

Reply:
Removed